web/src/components/core/Page.jsx

```diff
@@ -218,6 +248,7 @@ const Page = ({ icon, title = "Agama", children }) => {
       </footer>
 
       <Sidebar isOpen={sidebarOpen} onClose={closeSidebar} />
+      <If condition={issuesDialogOpen} then={<IssuesDialog onClose={closeIssuesDialog} />} />
```

Comment:
Maybe we can go for another strategy here: to keep the `IssuesDialog` always mounted but receiving a `isOpen` prop to know if it should be rendered or not. Similar to Popup/Modal. 

Not sure, honestly, but somehow it looks clean and better to me at first thought.


```suggestion
      <IssuesDialog isOpen={issuesDialogOpen}  onClose={closeIssuesDialog} />
```